### ISSUE TITLE
Scanning Tool for Amplica Access

### ISSUE THREAD

**Issue by p5150j** [LibertyDSNP/frequency#1399]
We need a small utility/service that scans for transfers to any account under the custodial wallet "control." And alert if there are any. (@p5150j is waiting on legal for full scope here)

### Acceptance criteria 
- it should send off an email if the wallet address' does any transfers 
- the email should have the pub key, the amount transferred and the address it was transferred to
- ...
- ...    

### Subtaks (each subtask can be its own ticket)
- task 1
- task 2 
- etc
